plugins/postcss-color-function/src/convert-prophoto-rgb-to-srgb.ts

```diff
@@ -15,6 +15,13 @@ export function prophotoRgbToSRgb(prophoto: color): color {
 	let oklch = conversion.slice() as color;
 	oklch = XYZ_to_OKLab(oklch);
 	oklch = OKLab_to_OKLCH(oklch);
+	if (oklch[0] < 0.000001) {
+		oklch = [0, 0, 0] as color;
+	}
+
+	if (oklch[0] > 0.999999) {
+		oklch = [1, 0, 0] as color;
+	}
```

Comment:
This is closer to 0 or 1 than to the smallest/largest possible value that can be expressed in a 16bit color system.